crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -801,6 +801,9 @@ en:
         failed_proposals:
           error: 'These proposals report deployment failures: %{proposals}.'
           help: 'Check the state of the specific barclamps and redeploy them correctly or disable failing proposals.'
+        unsaved_proposals:
+          error: 'These proposals contain nodes which were deleted: %{proposals}.'
```

Comment:
Are we sure that the case is only "nodes that were deleted" ? I think it might be safer to say something like "which are not known to crowbar"...